database/changesets/20210422_ContinuousWorkloadInstanceTable.xml

```diff
@@ -12,20 +12,33 @@
                id="20210422"
                logicalFilePath="20210422_ContinuousWorkloadInstanceTable.xml"
                runInTransaction="false">
-        <sql>
-            CREATE TYPE ContinuousWorkloadInstance AS (
-                id             bigint,      -- unique to this table
-                source_type    source,      -- source type
-                source_item    text,        -- id of the source instance
-                executor_type  executor,    -- executor type
-                executor_item  text,        -- id of the executor instance
-                sink_type      sink,        -- sink type
-                sink_item      text,        -- id of the sink instance
-                updated        timestamptz  -- status update time
-            )
-            <comment>
-                the ContinuousWorkloadInstance table schema
-            </comment>
-        </sql>
```

Comment:
The raw SQL is much more readable. What's the reason to prefer XML?

Reply:
Per offline discussion, XML is rollback-able by liquibase, but in this case, I found examples on liquibase docs for what I wanted to do in XML, so…

Reply:
And I will address this in a later commit!